### PR TITLE
rpc: add chunked genesis endpoint

### PR DESCRIPTION
**Pull request by tychoish**
reflects the changes in: https://github.com/tendermint/tendermint/pull/6445

### CODE REVIEW

File: spec/rpc/README.md [hunk 1]
```diff
@@ -848,6 +849,42 @@ curl -X POST https://localhost:26657 -d "{\"jsonrpc\":\"2.0\",\"id\":1,\"method\
 }
 ```
 
+### GenesisChunked
+
+Get the genesis document in a chunks to support easily transfering larger documents.
+
+#### Parameters
+
+- `chunkID`: the index number of the chunk that you wish to fetch.
```
Comment (cmwaters) [inline spec/rpc/README.md:858]: Personally, I think just calling it `chunk` would be better, but this is a super minor detail so I don't really mind.

Comment (cmwaters) [inline spec/rpc/README.md:858]:
```suggestion
- `chunkID (integer)`: the index number of the chunk that you wish to fetch.
```

**Review by cmwaters — APPROVED** [review]
LGTM 👍 

If a genesis file is greater than the size of one chunk then calling `/genesis` will return an error right? Do we also want to include a note of this in the `Genesis` section

File: spec/rpc/README.md (hunk 1 above)
Comment (alexanderbez) [reply spec/rpc/README.md:858]: It should also state if it's one-based or zero-based.

**Review by alexanderbez — APPROVED** [review]

**Comment by tychoish** [comment]
I have not changed the behavior of the genesis endpoint, yet, but we could do that.

**Review by cmwaters — APPROVED** [review]

**Review by marbar3778 — APPROVED** [review]
thanks for making this change

**Comment by marbar3778** [comment]
@tychoish can you fix the linter and then we can merge this

**Comment by marbar3778** [comment]
@tychoish can you fix the linting in this PR. Then we can merge

**Pull request merged**